src/main/java/graphql/language/AstZipper.java

```diff
@@ -0,0 +1,85 @@
+package graphql.language;
```

Comment:
Should all this transformation code be in `graphql.language` or one down in say `graphql.language.transformation` ?

I think the latter